dashboard/modules/stats_collector/stats_collector_consts.py

```diff
@@ -1,3 +1,3 @@
-NODE_STATS_UPDATE_INTERVAL_SECONDS = 1
+NODE_STATS_UPDATE_INTERVAL_SECONDS = 5
```

Comment:
I can keep this value 1 second, and in the future, I will make the value configurable.

If the cluster is large, then the 1 second interval is too short. And this PR enable cache for `GET /nodes` and `GET /node/{node id}` with default TTL is 60 seconds. Can you help me confirm that the `AIOHTTP_CACHE_TTL_SECONDS` value is appropriate?

Reply:
Yep totally understand we should make it configurable. I'd just like to keep the default for OSS because most of the users won't have scalability issues (and real time is always better). For the TTL, I'd like to suggest to find a way to track the cache hit (I don't have any opinion about the length of this because I don't have any sense of how often the values are changed).

Reply:
OK. I will keep the `NODE_STATS_UPDATE_INTERVAL_SECONDS` 1 second. 

About the TTL, for example, `node info` is made from `GcsNodeInfo`, `Node stats` and `Node physical stats`. The update time for them is,
| Data Source | Nodes(GcsNodeInfo) | Node Stats | Node Physical Stats |
|---|---| --- |---|
|Update By|head|stats_collector|reporter|
|Interval Seconds| 5 |  1(this one) | 2.5 |

So, the node info will be fully refreshed every 5 seconds. If the cache TTL <= 5 seconds, users will always get the refreshed data. I will reset the default TTL of cache to 2 (<=5) seconds to make the data real-time.